src/transformStateWithClones.js

```diff
@@ -65,7 +65,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
+  let stateClone = Object.assign({}, state);
```

Comment:
there You can use spread operator ... instead of Object.asign. Something like that: ``` let stateClone = { ... state }